src/Components/CroppedThumbnail/CroppedThumbnail.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import { Image, ImageProps, ImageURISource } from "react-native";
+
+type CropMode = "c_scale" | "c_fit" | "c_crop";
```

Comment:
`c_` here stands for crop. No need to dublicate this letter in the API of your component